cmd/main.go

```diff
@@ -157,7 +156,6 @@ func (opts *options) Parse() {
 	flag.BoolVar(&opts.introspectionEnabled, "introspectionEnabled", false, "Enable introspection (beta)")
 	flag.BoolVar(&opts.datadogAgentProfileEnabled, "datadogAgentProfileEnabled", false, "Enable DatadogAgentProfile controller (beta)")
 	flag.BoolVar(&opts.remoteConfigEnabled, "remoteConfigEnabled", false, "Enable RemoteConfig capabilities in the Operator (beta)")
-	flag.BoolVar(&opts.otelAgentEnabled, "otelAgentEnabled", false, "Enable the OTel agent container (beta)")
```

Comment:
Isn't this breaking change for staging?

Reply:
My understanding is that staging uses annotations, not the flag so this shouldn't be a breaking change (I left annotation support for now, until we update staging to use feature, in which case I will remove annotation support).

cc @levan-m to confirm.

Reply:
Yes, staging shouldn't be using this, instead [annotation](https://github.com/DataDog/k8s-datadog-agent-ops/blob/main/charts/datadog-agent-with-operator/templates/datadogagent.yaml#L13-L15) on DDA.